dev-requirements.txt

```diff
@@ -6,3 +6,4 @@ flake8==4.0.1
 black==22.3.0
 mypy==0.910
 types-PyYAML==6.0.4
+Pint==0.18
```

Comment:
You shouldn't need this in dev-requirements - it's in setup.py. There's a version conflict here which is making the tests fail.

If you need to install the package for dev you can use `pip install -e .` which will also pull in the install requirements.

Reply:
you fixed this properly in 90e49ecb - dropped this commit